go/src/koding/klientctl/main.go

```diff
@@ -261,6 +261,12 @@ func main() {
 					Name:  "force",
 					Usage: "Updates kd & klient to latest available version.",
 				},
+				cli.BoolFlag{
+					Name: "continue",
+					// TODO(leeola): Update to the latest cli package, and use the Hidden flag
+					// for this bool flag.
+					Usage: "An internal flag, not intended for use.",
```

Comment:
`Internal use only.` - make it consistent with metrics cmd

After `go1.6` branch is merged we should update deps and fix this, the `kd help` output gets hurt by this.